### ISSUE TITLE
Many albums display tracks multiple times

### ISSUE THREAD

**Issue by RanTalbott** [janbar/noson-app#205]
I haven't been able to spot a pattern for this behavior yet, but some albums list their tracks anywhere from 2 to 4 times (so far: could be more for some albums).

This makes it difficult to play albums: I have to queue up all the tracks, then manually delete each duplicate.

**Comment by janbar** [comment]
If some tracks are duplicated, that means you have duplicated files in your music directory.

**Comment by RanTalbott** [comment]
Nope. I just checked several albums and it now appears that **every** track is duplicated. Even the ones on the fake "albums" that noson creates by splitting up multi-artist albums.

This might, or might not, be related to the fact that the thrift store had a clearance sale, and I have now exceeded 1024 albums. It **probably** isn't, but I can't be sure because the duplication means that I've used it very little, so I don't have many samples to work from.
